src/test/java/com/ebivariation/contigalias/NCBIBrowserTest.java

```diff
@@ -49,13 +49,14 @@ void connect() throws IOException {
 
     @Test
     void navigateToAllGenomesDirectory() throws IOException {
-        ncbiBrowser.navigateToAllGenomesDirectory();
+        boolean b = ncbiBrowser.changeWorkingDirectoryToGenomesAll();
+        assertTrue(b);
```

Comment:
Please don't name variables like `b`, use meaningful names. Although in this case I don't think you need a variable
```suggestion
        assertTrue(ncbiBrowser.changeWorkingDirectoryToGenomesAll());
```